### PR TITLE
Doc infra updates

### PR DESCRIPTION
**Pull request by OriolAbril**
Once merged, the preview action will add a link at the bottom of the PRs with the link to the docs preview so it is easier to access. Also updates the version of the theme to use pypi (just created a package for the theme) and pins the version.

### CODE REVIEW

**Review by ricardoV94 — APPROVED** [review]

**Review by maresb — APPROVED** [review]
Thanks @OriolAbril!

We can also put this up on conda-forge. (If I do that, could I add you as a feedstock maintainer?)

**Comment by OriolAbril** [comment]
> We can also put this up on conda-forge. (If I do that, could I add you as a feedstock maintainer?)

Sounds good yeah, do you want to wait for this before merging?

**Review by twiecki — APPROVED** [review]

**Pull request merged**
